AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -335,7 +336,9 @@ - (void)endUpdatesWithCompletion:(void (^)(BOOL))completion
   _batchUpdateCounter--;
 
   if (_batchUpdateCounter == 0) {
-    [_delegate dataControllerBeginUpdates:self];
+    [_editingTransactionQueue addOperationWithBlock:^{
+      [_delegate dataControllerBeginUpdates:self];
```

Comment:
I've been out of this code for a week, but I'm nearly positive this will result in calling the delegate on a background queue, which is not currently intended / is likely to break the threading model currently being used in this class.